consumer/dragonboard820c/build/linux-kernel.md

```diff
@@ -7,6 +7,14 @@ permalink: /documentation/consumer/dragonboard820c/build/linux-kernel.md.html
 
 The following instructions can be used to build Linux Kernel from source for Dragonboard-820c.  Note this is cross-compiled on a host development system.   These instructions assume the host machine is Linux based.
 
+## Install Required Packages
+
+The following command installs packages which are required to build the
+kernel on Debian based systems:
+
+```shell
+$ sudo apt-get install git build-essential kernel-package fakeroot libncurses5-dev libssl-dev ccache
+```
```

Comment:
Note about other distros (from DB410C) is not present in this one.

Reply:
Added. Thanks